sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/postgres/PostgresInt4Tests.java

```diff
@@ -0,0 +1,419 @@
+                        " -2147483647 | -1073741823\n" +
+                        "(5 rows)\n" +
+                        "\n" +
+                        "SELECT -2+3 AS one;\n" +
+                        " one \n" +
+                        "-----\n" +
+                        "   1\n" +
+                        "(1 row)\n" +
+                        "\n" +
+                        "SELECT 4-2 AS two;\n" +
+                        " two \n" +
+                        "-----\n" +
+                        "   2\n" +
+                        "(1 row)\n" +
+                        "\n" +
+                        "SELECT 2- -1 AS three;\n" +
+                        " three \n" +
+                        "-------\n" +
+                        "     3\n" +
+                        "(1 row)\n" +
+                        "\n" +
+                        "SELECT 2 - -2 AS four;\n" +
+                        " four \n" +
+                        "------\n" +
+                        "    4\n" +
+                        "(1 row)\n" +
+                        "\n" +
+                        "SELECT '2'::INT2 * '2'::INT2 = '16'::INT2 / '4'::INT2 AS true;\n" +
+                        " true \n" +
+                        "------\n" +
+                        " t\n" +
+                        "(1 row)\n" +
+                        "\n" +
+                        "SELECT '2'::INT4 * '2'::INT2 = '16'::INT2 / '4'::INT4 AS true;\n" +
+                        " true \n" +
+                        "------\n" +
+                        " t\n" +
+                        "(1 row)\n" +
+                        "\n" +
+                        "SELECT '2'::INT2 * '2'::INT4 = '16'::INT4 / '4'::INT2 AS true;\n" +
+                        " true \n" +
+                        "------\n" +
+                        " t\n" +
+                        "(1 row)\n" +
+                        "\n" +
+                        "SELECT '1000'::INT4 < '999'::INT4 AS false;\n" +
+                        " false \n" +
+                        "-------\n" +
+                        " f\n" +
+                        "(1 row)\n" +
+                        "\n" +
+                        "SELECT 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 AS ten;\n" +
+                        " ten \n" +
+                        "-----\n" +
+                        "  10\n" +
+                        "(1 row)\n" +
+                        "\n" +
+                        "SELECT 2 + 2 / 2 AS three;\n" +
+                        " three \n" +
+                        "-------\n" +
+                        "     3\n" +
+                        "(1 row)\n" +
+                        "\n" +
+                        "SELECT (2 + 2) / 2 AS two;\n" +
+                        " two \n" +
+                        "-----\n" +
+                        "   2\n" +
+                        "(1 row)"
+        );
+    }
+
+    // Check PostgresInt2Tests::testSelectOverflow for details
+    @Test @Ignore
```

Comment:
why is this ignored?

Reply:
Ignored because we overflow and wrap around instead of throwing an error like Postgres.